module-4/metascan/metascan.py

```diff
@@ -0,0 +1,91 @@
+import os
+import csv
+import requests
+from bs4 import BeautifulSoup
+from urllib.parse import urljoin, urlparse
+from PyPDF2 import PdfReader
+import argparse
+
+def extract_pdf_links(url):
```

Comment:
better add the info from where you extract pdf links

--> `extract_pdf_links_from_website` (or webpage) - indicates that you will scrape HTML files in order to find PDFs in it.